services/src/main/java/org/jboss/windup/web/services/rest/RegisteredApplicationEndpoint.java

```diff
@@ -20,72 +20,32 @@
 {
```

Comment:
Yes. There are 2 endpoints - MigrationProjectRegisteredApplicationsEndpoint for registering new applications and RegisteredApplicationEndpoint for editing existing application.